pkg/apis/forklift/v1beta1/plan/vm.go

```diff
@@ -30,6 +30,10 @@ type VM struct {
 	ref.Ref `json:",inline"`
 	// Enable hooks.
 	Hooks []HookRef `json:"hooks,omitempty"`
+	// LUKS keys for virt-v2v
+	// References a secret containing disks decryption keys
```

Comment:
multi-line is not rendered nicely in the yaml and we see from the definition below that it references a Secret so how about replacing the two lines above with:
```suggestion
	// Disk decryption LUKS keys
```
?